owid/catalog/tables.py

```diff
@@ -232,3 +234,24 @@ def all_columns(self) -> List[str]:
         "Return names of all columns in the dataset, including the index."
         combined: List[str] = filter(None, list(self.index.names) + list(self.columns))  # type: ignore
         return combined
+
+    def update_metadata_from_yaml(
```

Comment:
Why not put this on `TableMeta` instead, as you did for `DatasetMeta`?

Reply:
Ah, yeah, `TableMeta` is only for the things shared across all variables, as it's implemented now